_posts/2021-10-22-eslint-v8.1.0-released.md

```diff
@@ -5,6 +5,11 @@ tags:
   - release
   - minor
 author: mdjermanovic
+original:
```

Comment:
We would not include this section on all blog posts, only the ones that were re-posted from elsewhere.

Reply:
I'll remove this section from the blog post